WizardExtensions/MSTestv2IntelliTestExtension/MSTestv2TestFramework.cs

```diff
@@ -127,7 +127,7 @@ public override TypeName ExpectedExceptionAttribute
             {
                 if (this.expectedExceptionAttribute == null)
                 {
-                    this.expectedExceptionAttribute = MSTestv2TestFrameworkMetadata.AttributeName("ExpectedExceptionAttribute");
+                    this.expectedExceptionAttribute = MSTestv2TestFrameworkMetadata.AttributeName("ExpectedException");
```

Comment:
Interesting. Do validate through the other pex scenarios before merging. @harshjain2 can help with that.